ui/src/StateContainer.js

```diff
@@ -132,14 +162,20 @@ export class StateContainer extends LitElement {
             @skip-prompt="${this.handleChangePrompt}"
             @end-session="${this.handleEndSession}"
             @update-wave="${this.handleUpdateWave}"
+            @update-toast="${this.handleUpdateToast}"
         >
             <vox-view-container
                 .view=${this.view}
                 ?can-record=${this.canRecord}
```

Comment:
How does the ? differ from the . & will they affect how these properties are passed down?

Reply:
pretty sure when you use ? instead of the . you are specifying it as a boolean attribute not boolean property

Reply:
^ elvis, boolean properties must be prefixed with ? as per litelement documentation